Sources/Workspace/Manifest.swift

```diff
@@ -0,0 +1,34 @@
+/*
+ This source file is part of the Swift.org open source project
+ 
+ Copyright (c) 2019 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+ 
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+ */
+
+import PackageModel
+import PackageLoading
+
+extension Manifest {
+
+    /// Loads a manifest from a package repository using the resources associated with a particular `swift` executable.
+    ///
+    /// - Parameters:
+    ///     - package: The absolute path of the package root.
+    ///     - swiftExecutable: The absolute path of a `swift` executable.
+    ///         Its associated resources will be used by the loader.
+    public static func loadManifest(
+        from package: AbsolutePath,
+        with swiftExecutable: AbsolutePath) throws -> Manifest {
+
+        let resources = try UserManifestResources(swiftExectuable: swiftExecutable)
```

Comment:
Let's move `UserManifestResources` to `PackageLoading` so we don't have to create an extension at the Workspace level.

Reply:
That is kinda what I would have preferred to do the first time around. I refrained in order to minimize code churn. Doing it now though, along with the rest of what you have asked for.

Reply:
Sank this into PackageLoading.

(Not pushed yet.)